config/http_config.go

```diff
@@ -743,44 +870,33 @@ func NewTLSRoundTripper(
 	}
 	t.rt = rt
 
-	_, t.hashCAFile, err = t.getCAWithHash()
```

Comment:
I might be missing something obvious here: the hash is kept so that changes in the certificate can be detected, but in order to compute the hash, the file has to be loaded each time, so the potential savings in using a hash to avoid the longer byte-for-byte comparison are lost (?) by having to read the entire certificate from disk and then compute the hash. Is it that _some_ changes in the file won't cause the hash to change (in other words, only a part of the file content's is used to compute the hash)?